### DISCUSSION TITLE
Installing SLEAP on Colab- Imgaug Error

### DISCUSSION THREAD

**Discussion by LA120** [talmolab/sleap#799] [Help!]
Hello,

I am trying to run the training and inference using colab instead of the GPU as my device does not have one. However, when I try to install sleap onto colab it is consistently giving this error:

"ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
albumentations 0.1.12 requires imgaug<0.2.7,>=0.2.5, but you have imgaug 0.4.0 which is incompatible."

I have tried to install imgaug's different versions but whenever I uninstall 0.4.0 and install 0.2.7 , it says I need 0.4.0 and vice versa.  I will attach an image of my issue. Please advise. 

![Screen Shot 2022-06-20 at 10 07 03 AM](https://user-images.githubusercontent.com/107879091/174650988-cb3bfb45-3138-4cee-9c2d-a4692414e806.png)
![Screen Shot 2022-06-20 at 10 07 11 AM](https://user-images.githubusercontent.com/107879091/174650991-99daca95-627b-4faa-8a7d-c40ac5e739c0.png)
![Screen Shot 2022-06-20 at 10 07 21 AM](https://user-images.githubusercontent.com/107879091/174650995-7634d9ea-e172-448a-b1f7-76ae6a1867cb.png)

**Comment by roomrys** [comment]
Hi @LA120,

Running the installation commands
```
!pip uninstall -y opencv-python opencv-contrib-python
!pip install sleap
```
will throw the following ERROR
```
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
albumentations 0.1.12 requires imgaug<0.2.7,>=0.2.5, but you have imgaug 0.4.0 which is incompatible.
```
and while we are planning on making the switch to `albumentations` (see: Switch to albumentations for augmentation #648), we are currently using the `imgaug` package for image augmentation. 

Feel free to continue with the rest of the notebook as this should not affect training.

Thanks,
Liezl
